### PR TITLE
Add new member badges

### LINKED ISSUE #156: Show badge on member profile
We should show the "Open Source Pledge Member" badge on member profiles.

For those who join before Oct 8 the badge variant should be "Open Source Pledge Innovator."

### PR DESCRIPTION
**Pull request by vladh**
This PR also contains a new JSON lockfile system, and some documentation.

Closes #156.

### CODE REVIEW

File: bin/update-member [hunk 1]
```diff
@@ -12,15 +12,15 @@ Must be run in the repo's root.`;
 
 
 async function updateMember(name, url) {
-  // NOTE: Temporary hack
-  if (name == 'pixee') {
-    return;
-  }
```
Comment (chadwhitacre) [inline bin/update-member:18]: haha

File: src/layouts/Layout.astro [hunk 2]
```diff
@@ -259,7 +259,7 @@ const { title, navless } = Astro.props;
     button {
       display: inline-block;
       padding: 0.4rem 0.8rem;
-      color: var(--color-bg);
+      color: black;
```
Comment (chadwhitacre) [inline src/layouts/Layout.astro:262]: Are we really using pure black? Surprised if so.

Comment (vladh) [reply src/layouts/Layout.astro:262]: Me too — the text colour in yesterday's badge brief was `#000000`. Shall I change it to `--color-bg`?

**Comment by chadwhitacre** [comment]
Lol posted a screenshot from localhost before merging 🤪

https://x.com/chadwhitacre_/status/1841475765407211922
https://www.linkedin.com/feed/update/urn:li:activity:7247243090288136193/

**Pull request merged**